src/readline.rs

```diff
@@ -96,11 +97,14 @@ impl Completer for RLHelper {
 }
 
 impl Hinter for RLHelper {
-    fn hint(&self, line: &str, a: usize, b: &Context) -> Option<String> {
+    type Hint = String;
+    fn hint(&self, line: &str, a: usize, b: &Context) -> Option<Self::Hint> {
```

Comment:
Why do we need this change?

Reply:
Otherwise I get the following error from the compiler:

```
   Compiling eva v0.2.7 (/home/ma27/Projects/eva)
error[E0046]: not all trait items implemented, missing: `Hint`
  --> src/readline.rs:99:1
   |
99 | impl Hinter for RLHelper {
   | ^^^^^^^^^^^^^^^^^^^^^^^^ missing `Hint` in implementation
   |
   = help: implement the missing item: `type Hint = Type;`

For more information about this error, try `rustc --explain E0046`.
error: could not compile `eva` due to previous error

```

Reply:
What I mean is why not keep `-> Option<String>`?

Reply:
I also got the error with `-> Option<String>`.